projects/tidb/build.sh

```diff
@@ -15,11 +15,13 @@
 #
 ################################################################################
 
-# Insert empty main function
-sed -i '23 i\func main(){}'\\n $SRC/tidb/plugin/conn_ip_example/conn_ip_example.go
+if [[ $SANITIZER = *coverage* ]]; then
+        compile_go_fuzzer github.com/pingcap/tidb/types FuzzMarshalJSON fuzzMarshalJSON
```

Comment:
@catenacyber @AdamKorcz - why is this happening, is something up with coverage that we need to give full path to compile fuzzer

Reply:
It may be that `compile_go_fuzzer` requires the golang package name in order to get coverage for the whole module : we will have coverage for whole `github.com/pingcap/tidb` and not just `github.com/pingcap/tidb/types`

Does `compile_go_fuzzer github.com/pingcap/tidb/types FuzzMarshalJSON fuzzMarshalJSON` not work for SANITIZER=address ?

Reply:
@catenacyber 
Unfortunately not. We get the following error:
```
no required module provides package github.com/pingcap/tidb/types/types; to add it:
        go get github.com/pingcap/tidb/types/types
```

Reply:
Works locally for me
I see
```
go: downloading github.com/jmespath/go-jmespath/internal/testify v1.5.1
github.com/pingcap/tidb/types
github.com/pingcap/tidb/types
Running go-fuzz -tags gofuzz -func FuzzMarshalJSON -o fuzzMarshalJSON.a github.com/pingcap/tidb/types
```
Did you type `types` twice in `compile_go_fuzzer github.com/pingcap/tidb/types FuzzMarshalJSON fuzzMarshalJSON`?

Reply:
Thanks for the heads up. I have submitted a PR.